spec/features/bulk_updates/bulk_updates_data_centers_spec.rb

```diff
@@ -331,7 +347,7 @@
                 expect(page).to have_no_content('ESA/ED')
 
                 expect(page).to have_content('AARHUS-HYDRO', count: 2)
-                expect(page).to have_content('Hydrogeophysics Group, Aarhus University ', count: 2)
+                expect(page).to have_content('Hydrogeophysics Group, Aarhus University', count: 2)
```

Comment:
Why did the old version have a space at the end?   Just a typo or was it on purpose?

Reply:
The KMS (both in the local CMR and the other environments) has this trailing whitespace in the long name keyword. I brought this up with @jceaser, you can ask him about tickets and what the plan is for it. What it means for our tests is that previously it was rendered, but with the change in webdriver and headless browser, it's not rendered or picked up anymore so needs to be removed. However, when it is in an input field, because it's being populated from the controlled keywords, the space is still there.